assets/js/components/AnalyticsInactiveCTA.js

```diff
@@ -24,64 +24,56 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { Fragment } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import {
-	activateOrDeactivateModule,
-	getReAuthURL,
 	showErrorNotification,
 } from '../util';
-import { refreshAuthentication } from '../util/refresh-authentication';
-import data from './data';
+import Data from 'googlesitekit-data';
 import CTA from './notifications/cta';
 import GenericError from './notifications/generic-error';
+import { STORE_NAME as USER_STORE, PERMISSION_MANAGE_OPTIONS } from '../googlesitekit/datastore/user/constants';
```

Comment:
I realize this was named this way in the IB but please use the same convention we use for store name aliases everywhere else.
```suggestion
import { STORE_NAME as CORE_USER, PERMISSION_MANAGE_OPTIONS } from '../googlesitekit/datastore/user/constants';
```